### ISSUE TITLE
Remove unused "Script Console" resource messages

### ISSUE THREAD

**Issue by thc202** [zaproxy/zaproxy#2386]
The following resource messages (from [Messages.properties](https://github.com/zaproxy/zap-extensions/blob/3e95c034fbb66c2e763492367904323efe99f57e/src/org/zaproxy/zap/extension/scripts/resources/Messages.properties#L45-L47)) are not used by the add-on but by current main releases of ZAP core:
```Properties
scripts.interface.active.error			= The script does not correctly implement the required interface.
scripts.interface.passive.error			= The script does not correctly implement the required interface.
scripts.interface.targeted.error		= The script does not correctly implement the required interface.
```
(not all the resource messages might be in use in latest ZAP version though)

Changes done in #2167 removed its use, so they can be safely deleted once "Script Console" add-on targets ZAP v2.5.0 (expected version to include the changes).

The add-on currently targets versions >= 2.4.0.

**Comment by thc202** [comment]
Fixed in zaproxy/zap-extensions#552.

**Comment by thc202** [comment]
Released in version 18 of Script Console add-on (bundled in ZAP 2.6.0).
